Makefile

```diff
@@ -9,7 +9,7 @@ BUILD_BASE_DIR ?= $(PWD)
 GOTMP=.gotmp
 SHELL = /bin/bash
 PWD = $(shell pwd)
-GOFILES = $(shell find $(SRC_DIRS) -type f)
+GOFILES = $(shell find $(SRC_DIRS) -type f ! -path "*/testdata/*")
```

Comment:
There was a problem on `make build`:

https://github.com/ddev/ddev/actions/runs/10881287727/job/30189913793?pr=6406#step:16:29
```
tangle cmd/ddev/cmd/testdata/TestCmdAddonGetWithEnvironmentVariablesFromUnknownFlags/env_files/.env.web
/bin/bash: line 1: tangle: command not found
make: *** [<builtin>: cmd/ddev/cmd/testdata/TestCmdAddonGetWithEnvironmentVariablesFromUnknownFlags/env_files/.env.p] Error 127
```

The reason for this behavior:

https://www.gnu.org/software/make/manual/html_node/Catalogue-of-Rules.html

> TeX and Web
> n.dvi is made from n.tex with the recipe ‘$(TEX)’. n.tex is made from n.web with ‘$(WEAVE)’, or from n.w (and from n.ch if it exists or can be made) with ‘$(CWEAVE)’. n.p is made from n.web with ‘$(TANGLE)’ and n.c is made from n.w (and from n.ch if it exists or can be made) with ‘$(CTANGLE)’.

Excluding all `/testdata/` files from the `Makefile` helped.

Thanks to @rfay for showing me the rules for `make`.